migrate/20250819_add_kubernetes_node_model.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:kubernetes_node) do
+      column :id, :uuid, primary_key: true
+      column :created_at, :timestamptz, null: false, default: Sequel::CURRENT_TIMESTAMP
+      foreign_key :vm_id, :vm, type: :uuid, null: false
+      foreign_key :kubernetes_cluster_id, :kubernetes_cluster, type: :uuid, null: false
```

Comment:
no action needed: not related to this PR, but it feels like if CP nodes would also belong to a nodepool, that could make things more neat in the codebase. Maybe something to do later.